gradle.properties

```diff
@@ -7,6 +7,6 @@ org.gradle.jvmargs=-Xmx2g -Dfile.encoding=UTF-8
 kotlinVersion=1.9.10
 nativeBuildToolsVersion=0.9.27
 springFrameworkVersion=6.1.0-SNAPSHOT
-tomcatVersion=10.1.13
+tomcatVersion=10.1.14
```

Comment:
This breaks some tests, which I haven't fixed in this PR. I had to upgrade the Tomcat version to be able to call `protocol.addSslHostConfig(sslHostConfig, true);`